dz6/dz6/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace Task6
+{
+    internal class Program
+    {
+        public string Relanguage(string text)
+        {
```

Comment:
Перевод в верхний регистр лучше делать здесь.
```suggestion
        public string Relanguage(string text)
        {
            text = text.ToUpper();
```